src/scrapper/database/dbs/create_db_sqlite3.sql

```diff
@@ -48,6 +48,7 @@ CREATE TABLE `course_unit` (
   `year` smallint(6) NOT NULL,
   `schedule_url` varchar(2000) DEFAULT NULL,
   `last_updated` datetime NOT NULL,
+  `hash` varchar(32) ,
```

Comment:
Can you confirm this enough? or we need `varchar(64)` ?

Reply:
@jose-carlos-sousa noticed that this is wrong and he corrected it on his pr on the backend, but it wasn't fixed in this schema.

Yes, we need 64.

Reply:
Thank you for reminding me to update it, I have done so

Reply:
Nice!